build.sbt

```diff
@@ -158,7 +158,7 @@ lazy val `support-workers` = (project in file("support-workers"))
     mergeStrategySettings,
   )
   .dependsOn(
-    `support-services`,
+    `support-services` % "test->test;compile->compile",
```

Comment:
needed so `CreateZuoraSubscriptionSupporterPlusStepsSpec` can reference `CatalogServiceSpec.serviceWithFixtures` which has a copy of the PROD catalog.  It looks like there are already 3(!) copies of the prod catalog in this repo, presumably because other people hit this same issue, but I didn't want even more copy and paste as it's wasteful.